Filtered_kernel/include/CGAL/Lazy.h

```diff
@@ -1594,7 +1602,7 @@ struct Lazy_construction_variant {
     }
     CGAL_BRANCH_PROFILER_BRANCH(tmp);
     Protect_FPU_rounding<!Protection> P2(CGAL_FE_TONEAREST);
-
+    CGAL_expensive_assertion(FPU_get_cw() == CGAL_FE_TONEAREST);
```

Comment:
Why not use Checked_protect_FPU_rounding (as in Interval_nt) which is supposed to do the same as those 2 lines? (although its last assertion seems to be bogus)